Tasks/Common/packaging-common/util.ts

```diff
@@ -78,6 +78,10 @@ export function toNerfDart(uri: string): string {
 
 export function getProjectAndFeedIdFromInputParam(inputParam: string): any {
```

Comment:
Since you made a change here you'll need to bump the versions of all the packaging tasks - there's a script in the packaging-common folder to help with this

Reply:
Thanks, bumped all the affected task versions.  Thanks also for the handy script :)